### ISSUE TITLE
Calendar component accessibility issue with timeControlType="select"

### ISSUE THREAD

**Issue by mertsincan** [primefaces/primefaces#3937]
Reported By PRO User;
```
Description

Select (combo) control does not have an explicit label or a title attribute.
Status

Source 
------------
p:calendar (with time selection enabled)

Generated code 
------------------------------------
<select class="ui-timepicker-select ui-state-default ui-corner-all" data-unit="hour" data-min="0" data-max="23" data-step="1">
<option value="0">
12 AM
</option>
<option value="1">
01 AM
</option>
<option value="2">
02 AM
</option>
<option value="3">
03 AM
</option>
<option value="4">
04 AM
</option>
<option value="5">
05 AM
</option>
<option value="6">
06 AM
</option>
<option value="7">
07 AM
</option>
<option value="8">
08 AM
</option>
<option value="9">
09 AM
</option>
<option value="10">
10 AM
</option>
<option value="11">
11 AM
</option>
<option value="12">
12 PM
</option>
<option value="13">
01 PM
</option>
<option value="14">
02 PM
</option>
<option value="15">
03 PM
</option>
<option value="16">
04 PM
</option>
<option value="17">
05 PM
</option>
<option value="18">
06 PM
</option>
<option value="19">
07 PM
</option>
<option value="20">
08 PM
</option>
<option value="21">
09 PM
</option>
<option value="22">
10 PM
</option>
<option value="23">
11 PM
</option>
</select>

Standard(s) associated with this issue
---------------------------------------------------
WCAG 2, Level A, 1.3.1 - Info and Relationships

Section 508, Paragraph N - Electronic Forms
```
